api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/ListChildrenOnARTandFormulationsDataset.java

```diff
@@ -218,7 +218,7 @@ private DataDefinition getAge() {
     return spdd;
   }
 
-  private DataDefinition getNID() {
+  public DataDefinition getNID() {
```

Comment:
Currently this column is NOT specifying the identifier type, that need to be provided for the correct  identifier value. As things stand now, any identifier will be pulled. See how I did that https://github.com/jembi/openmrs-module-eptsreports/blob/MEPTS-903-Data_Quality_Report_to_Identify_Duplicate/api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/DQRForDuplicateFichaResumoDataSet.java#L123-L139, you might want to use that or make changes to your code to include those

Reply:
Once ticket 903 is not merged, I made changes on the code to include this identifier value.